docker/gaffer-kerberos/.env

```diff
@@ -0,0 +1,22 @@
+GAFFER_VERSION=2.0.0-alpha-0.3.1
+GAFFER_TOOLS_VERSION=2.0.0-alpha-0.3.1
+GAFFER_PRINCIPAL=gaffer
+GAFFER_KRB_PASSWORD=change-password-for-gaffer
+
+ZOOKEEPER_VERSION=3.7.1
+ZOOKEEPER_PRINCIPAL=zookeeper
+ZOOKEEPER_KRB_PASSWORD=change-password-for-zookeeper
+
+ACCUMULO_VERSION=2.0.1
+ACCUMULO_CONF_DIR=/etc/accumulo/conf
+ACCUMULO_PRINCIPAL=accumulo
+ACCUMULO_KRB_PASSWORD=change-password-for-accumulo
+
+HADOOP_VERSION=3.3.3
+HADOOP_CONF_DIR=/etc/hadoop/conf
+HADOOP_PRINCIPAL=hadoop
+HADOOP_KRB_PASSWORD=change-password-for-hadoop
+
+REALM=GAFFER.DOCKER
```

Comment:
This seems to be hardcoded in a couple of places, like `kdc/entrypoint.sh`

Reply:
I added this when I started work as it seemed like it would good if it could be changed. But it ended up needing to be used in various config files which means it is hardcoded most of the time. I will remove it from here and make it always hardcoded, as otherwise it will look like it can be changed when this isn't true.